edx/edx-enterprise
# ENT-142: Delicious hand-picked documentation

**This PR is incomplete - do NOT merge it just yet.**

**Description:** This PR adds developer and admin documentation.

**JIRA:** [ENT-142](https://openedx.atlassian.net/browse/ENT-142)

**Testing instructions:**

If you have not already done so, create/activate a `virtualenv`_. Unless otherwise stated, assume all terminal code below is executed within the virtualenv.

    $ make docs

*Observe*

1. No docs errors in console
2. Browser is opened at an index page
3. `Getting started` section contains instructions on setting up edx-enterprise in production and development environments and also instructions on common development tasks (i.e. running tests, checking quality, etc.)
4. `Using edx-enterprise` section contains instruction on how to configure and use edx-enterprise app.

**Reviewers:**
- [x] @bradenmacdonald and/or @pomegranited 
- [ ] @grantgoodmanedX 

**Merge checklist:**
- [x] ~~Version bumped~~ No code changes - only docs.
- [x] ~~Changelog record added~~ No code-changes - only docs.
- [x] ~~Documentation updated (not only docstrings)~~ This **is** documentation PR
- [x] ~~Translations are updated~~ No code-changes - only docs.

**Post merge:**
      finished.
